sources/install.sh

```diff
@@ -960,6 +960,16 @@ function wireshark_sources() {
   wireshark.tar.xz
 }
 
+function trilium() {
```

Comment:
Can you add a call to this function in `install_tools` so that trilium actually gets installed?